hi-ml-histopathology/src/histopathology/configs/classification/DeepSMILECrck.py

```diff
@@ -40,8 +39,6 @@ def __init__(self, **kwargs: Any) -> None:
             is_finetune=False,
             is_caching=True,
             num_top_slides=0,
-            # declared in DatasetParams:
-            local_datasets=[Path("/tmp/datasets/TCGA-CRCk")],
             azure_datasets=["TCGA-CRCk"],
```

Comment:
can we remove hardcoded strings? 
`from histopathology.datasets.default_paths import TCGA_CRCK_DATASET_ID`

Reply:
The default path is still used in some tests, will leave that as is.